service-base/src/main/java/org/eclipse/hono/service/management/credentials/CommonCredential.java

```diff
@@ -108,6 +108,5 @@ public void checkValidity() {
         if (this.authId == null || this.authId.isEmpty()) {
             throw new IllegalStateException("missing auth ID");
         }
-        getSecrets().forEach(CommonSecret::checkValidity);
     }
```

Comment:
Javadoc needs to be updated here (removing the `This also calls {@link CommonSecret#checkValidity()} for all secrets.` part).

Reply:
fixed